### DISCUSSION TITLE
Microphone plugin missing from unpkg.com

### DISCUSSION THREAD

**Discussion by JayJayNinetyFive** [katspaugh/wavesurfer.js#3183] [General]
Hi guys!

I've been using the microphone plugin in one of my projects and now suddenly I get a 404 from the unpkg URL as it's removed. Could someone please point me towards a solution? Preferably the wavesurfer.microphone.min.js (7.2.2 or closer to that) file if you have it with you?

URL: https://unpkg.com/wavesurfer.js@7.2.2/dist/plugin/wavesurfer.microphone.min.js

Thanks a lot!!

**Comment by katspaugh** [comment]
It’s available only in v6: https://unpkg.com/wavesurfer.js@6/dist/plugin/wavesurfer.microphone.min.js

**Reply by JayJayNinetyFive** [reply]
Thanks!! It worked!
